wss/1.0/modules/ROOT/pages/validate-wss-examples.adoc

```diff
@@ -0,0 +1,462 @@
+= Validate Operation Examples
+
+This `validate` operation validates an incoming SOAP request that contains WSS information. The WSS module validates the request against a `WssInboundConfig` configuration.
+
+.Parameters
+[%header%autowidth.spread,cols="a*"]
+|===
+| Name | M | ES | Default Value | Type | Description
+| `request` | Yes | No | - | `TypedValue <InputStream>` | SOAP request
+| `version` | No | No | `SOAP_12` | `SoapVersion` | SOAP protocol version
+|===
+
+Output::
+The output is the same SOAP request received. The operation does not return any attribute.
```

Comment:
I don't understand the first sentence: "output is the same SOAP request received"?
Also, "any attributes":
```suggestion
The output is the same SOAP request received. The operation does not return any attributes.
```